internal/server/v1beta1/user_test.go

```diff
@@ -31,35 +31,74 @@ func TestGetUserStarredAssets(t *testing.T) {
 		size     = 10
 	)
 	type testCase struct {
+		UserUUID     string
+		UserID       string
 		Description  string
 		ExpectStatus codes.Code
+		WantErr      error
 		Setup        func(context.Context, *mocks.StarService)
 		PostCheck    func(resp *compassv1beta1.GetUserStarredAssetsResponse) error
 	}
 
 	testCases := []testCase{
 		{
+			UserUUID:     "",
+			UserID:       "",
```

Comment:
Not sure it makes sense to call `handler.GetUserStarredAssets` with this `UserID`.

Reply:
Yes, I am not sure if this helps will testing any real scenario. The idea of adding this was to cover some particular code lines.

Shall I remove this?

Reply:
Maybe add a different test altogether to cover specific scenario. Fitting into existing test cases does not work well

Reply:
I mean they are for testing some logic for this call itself.
What I am not sure is, if this situation will every arise - if it represent a useful/real scenario.